README.md

```diff
@@ -11,17 +11,15 @@ oreboot is mostly written in Rust, with assembly where needed.
 oreboot currently only plans to support LinuxBoot payloads.
 
 
-Demo
-----
-
-Oreboot+QEMU for RISC-V HiFive Unleased:
-
-[![asciicast](https://asciinema.org/a/XnWkMWTABuajsbGPMMTefjuZ2.svg)](https://asciinema.org/a/XnWkMWTABuajsbGPMMTefjuZ2)
+Demos
+-----
 
-Oreboot+QEMU for ARM:
+- [oreboot for ARM in QEMU](https://asciinema.org/a/Ne4Fwa4Wpt95dorEoVnHwiEkP)
+- [oreboot for RISC-V HiFive Unleased in QEMU](https://asciinema.org/a/XnWkMWTABuajsbGPMMTefjuZ2)
 
-[![asciinema](https://asciinema.org/a/Ne4Fwa4Wpt95dorEoVnHwiEkP.png)](https://asciinema.org/a/Ne4Fwa4Wpt95dorEoVnHwiEkP)
+Screenshot from RISC-V demo:
 
+![RISC-V QEMU](Documentation/img/qemu-riscv-demo.png)
```

Comment:
Would it be possible to make this an SVG or copy in the text instead to avoid boating the git repo too much?

Reply:
Definitely, then I'll go for the latter. I just took a screenshot.